ocfweb/tv/templatetags/tv_formatting.py

```diff
@@ -0,0 +1,14 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def tv_lab_hours(hour):
+    if hour:
+        if hour.open.minute != 0 or hour.close.minute != 0:
+            fmt = '{:%-I:%M%P}-{:%-I%M%P}'
+        else:
+            fmt = '{:%-I%P}-{:%-I%P}'
```

Comment:
So after testing this, hyphens don't wrap and cause the font size to scale according to the viewport width. en-dashes and m-dashes will split before/after, making the text wrap. This reduces visibility because it will show half the hours on another line, and if there are more than one group of hours, i.e. 12-4, 5-7 the second block will be pushed below the fold.